pkg/cmd/admin/policy/policy.go

```diff
@@ -47,6 +48,9 @@ func NewCommandPolicy(name, fullName string, f *clientcmd.Factory, out io.Writer
 
 func runHelp(cmd *cobra.Command, args []string) {
 	cmd.Help()
+
+	// make sure that we exit with non-zero status so that typoed commands don't look like they were successful
+	os.Exit(1)
```

Comment:
same question here, does this return `1` in cases where you just ran the base command?